### PR TITLE
Initialize stream

### PR DESCRIPTION
**Pull request by pdeziel**
This PR updates the protocol buffers and client code to be compatible with the new Ensign stream initialization scheme.

I have made the following changes:

1. Rebuilt the protocol buffers
2. Updated the gRPC client to send the open stream message and publish and subscribe.
3. Added some helper methods to wrap events into event wrappers for compatibility with Ensign.

### TODOs and questions

Still to do:

- [ ] Verify that the live tests pass against staging.

### CHECKLIST

- [x] _Is the commit message formatted correctly?_

- [x] _Do all of your functions and methods have docstrings?_
- [x] _Have you added/updated unit tests where appropriate?_
- [x] _Have you run the unit tests using `pytest`?_
- [x] _Is your code style correct (are you using PEP8, pyflakes)?_

### CODE REVIEW

File: pyensign/connection.py
Comment (bbengfort) [inline pyensign/connection.py:1]: Looks great! I guess we still have to figure out how to keep the pub stream open - but that's a later problem. The sub stream should stay open.

File: pyensign/exceptions.py [hunk 1]
```diff
@@ -135,9 +135,9 @@ class EnsignRPCError(EnsignError):
     pass
 
 
-class EnsignResponseType(EnsignError):
+class EnsignTypeError(EnsignError, TypeError):
```
Comment (bbengfort) [inline pyensign/exceptions.py:138]: 👍🏽

**Review by bbengfort — APPROVED** [review]
Excellent; sorry for all the hassle with the changed protobufs!

**Review by bbengfort — APPROVED** [review]
Excellent; sorry for all the hassle with the changed protobufs!

File: pyensign/api/v1beta1/event.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+def unwrap(event_wrapper):
+    """
+    Unwrap an event from an EventWrapper.
+    """
+
+    if event_wrapper.event is None or len(event_wrapper.event) == 0:
+        raise AttributeError("event wrapper contains no event")
+
+    event = event_pb2.Event()
+    event.ParseFromString(event_wrapper.event)
```
[the remark this answers is not in the document]
Comment (pdeziel) [reply pyensign/api/v1beta1/event.py:31]: I was looking around for that too - but it does return the binary string according to the [docs](https://googleapis.dev/python/protobuf/latest/google/protobuf/message.html#google.protobuf.message.Message.SerializeToString). According to the answers [here](https://stackoverflow.com/questions/1859438/using-python-how-do-i-get-a-binary-serialization-of-my-google-protobuf-message) they just haven't bothered to update the method name.

**Pull request merged**